_packaging/documentation.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Documentation"
```

Comment:
example stub page! this particular one has a list of content pages in the guide. tutorials are handled individually